plugins/node/opentelemetry-instrumentation-fastify/package.json

```diff
@@ -49,7 +49,7 @@
     "@fastify/express": "^2.0.2",
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/context-async-hooks": "^1.8.0",
-    "@opentelemetry/instrumentation-http": "0.45.0",
+    "@opentelemetry/instrumentation-http": "0.45.1",
```

Comment:
Similar here, why is this pinned but not others?
Again likely not a big deal as it is in the dev-deps

Reply:
Having this be a caret version would be better, I agree. I'll open a PR to change both.

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1784